fhir-server-webapp/pom.xml

```diff
@@ -79,6 +79,11 @@
             <artifactId>fhir-operation-versions</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-operation-bench</artifactId>
```

Comment:
i'm not sure if we should have this in our server by default.  maybe keep in fhir-parent but remove from here?  do any e2e tests in the CI pipeline depend on it?

Reply:
Right. We certainly wouldn't want it exposed by default. I'll take it out for now.